packages/@okta/vuepress-site/docs/reference/api/user-types/index.md

```diff
@@ -0,0 +1,432 @@
+```
+
+##### Response Example
+
+
+```http
+HTTP/1.1 204 No Content
+```
+
+## Specify the User Type of a New User
+
+The [Create User](/docs/reference/api/users/#create-user-with-non-default-user-type) operation accepts a type specification as part of the request body. The specification is a map, but currently the only key permitted is "id". The type specification is also added to the [User Model](/docs/reference/api/users/#user-model) (see [User object updates](#user-object-updates)), but after user creation the type is read-only.
+
+##### Example
+
+```json
+  "type": {
+    "id": "otyfnjfba4ye7pgjB0g4"
+  }
+```
+
+## User Schema Operation Extensions to Non-Default Types
+
+All operations documented under [User Schema Operations](/docs/reference/api/schemas/#user-schema-operations) will continue to operate as before. All these APIs now have an alternate form - replace the trailing `/default` with a schema ID - which will perform the relevant operation on the schema associated with any type.
+
+Just as a schema is currently associated with a User Type, the schema ID is related to the type ID: replace the leading `oty` with `osc`. For example, a User Type with ID `oty1234567890abcdefg` would have a Schema ID of `osc1234567890abcdefg`.
+
+In the future the linkage between Schema and User Type may be extended (for example, to allow multiple Types to share a Schema) but for now this is a 1:1 relationship.
+
+### Example
+
+With the existing Schemas API without User Types, if you want to add a new property to the schema you would make a POST to `/api/v1/meta/schemas/user/default` (updating the default because that is the only type).
+
+With the User Types API, if you created a new type `oty1234567890abcdefg` and later wish to add a property to its schema, send a POST to the same endpoint, replacing `default` with the appropriate Schema ID: `/api/v1/meta/schemas/user/osc1234567890abcdefg`.
+
+Everything else - body of the payload, return values, etc. - is the same as the [Schemas API](/docs/reference/api/schemas/).
+
+## Linked Object endpoint updates
+
+Linked Objects now apply to every User Type. If an org has the default type and 2 custom types, a Linked Object will be capable of linking a User of any of those 3 types to any other User. Each type sees exactly the same Linked Object definitions.
+
+As such, the reference to `/default` in the Linked Objects URL is now deprecated. All old URLs from the [Linked Objects API](/docs/reference/api/linked-objects/) will continue to work for backwards compatibility purposes. But the operations now reside at the new URLs (formed by removing `/default`).
+
+Deprecated URL: `/api/v1/meta/schemas/user/default/linkedObjects`
+New URL: `/api/v1/meta/schemas/user/linkedObjects`
+
+### Example
+
+To create a Linked Object Definition, you can continue to POST to `/api/v1/meta/schemas/user/default/linkedObjects`.
+
+You should POST to the new URL `/api/v1/meta/schemas/user/linkedObjects` instead.
+
```

Comment:
These two sentences are not clear to me. Is it that one URL is recommended and the other deprecated? Could you say: "To create a Linked Object Definition, make a POST request to the new URL, `/api/v1/meta/schemas/user/linkedObjects`  (the old URL, `/api/v1/meta/schemas/user/default/linkedObjects`, is deprecated)."

Reply:
Again, this section will be reworked for OKTA-248776.